test/source-test.js

```diff
@@ -21,3 +22,31 @@ test('resolve() converts markdown to html', t => {
     t.truthy(result[0].endsWith('.html'))
   })
 })
+
+test('resolve() includes custom css file', async t => {
+  t.plan(1)
+
+  var result = await source.resolve(['./README.md'], { css: 'custom.css' })
+  var data = await fs.readFile(result[0], 'utf-8')
+
+  t.regex(data, /.*custom\.css.*/g)
+})
+
+test('resolve() includes multiple custom css files', async t => {
+  t.plan(2)
+
+  var result = await source.resolve(['./README.md'], { css: ['first.css', 'second.css'] })
+  var data = await fs.readFile(result[0], 'utf-8')
```

Comment:
it looks like files are being created by the tests.  If that is the case can you make sure they get removed in a teardown step?